patternfly/patternfly-elements
# fix: Patch Jump links error

## pfe-jump-links

A few missing fallback attributes in the properties for Jump links nav and panel were causing regressions in production.

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Demo](https://deploy-preview-1224--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/) 


### What has changed and why

- `pfe-band`: Remove flex styles on the `.pfe-band__aside` region because it wasn't in a flex container; substitute `height: 100%` to get the aside to fill its parent.
- `pfe-jump-links-nav`: Add alias'ed attributes for autobuild and horizontal.
- `pfe-jump-links-panel`: Add alias'ed attributes for offset and scrolltarget; convert `this.menu_links` to a node list rather than an array and wrap the matches actions in an if statement.


### Testing instructions

1. Test jump links inside the aside region of a band (sticky functionality should work). [Link](https://deploy-preview-1224--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/#pfeband)
2. Test jump links with fallback attributes used. [Link](https://deploy-preview-1224--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/#pfeband)
3. Check for regressions in the pfe-band aside region now that it is filling 100% height. [Link](https://deploy-preview-1224--happy-galileo-ea79c4.netlify.app/elements/pfe-band)


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
